src/components/FlashNotification.vue

```diff
@@ -6,7 +6,7 @@ const { items } = useFlashNotification();
 
 <template>
   <div
-    class="fixed left-0 right-0 bottom-0 z-50 flex flex-col items-center mb-4 space-y-2 pointer-events-none"
+    class="flash-notification fixed left-0 right-0 bottom-0 flex flex-col items-center mb-4 space-y-2 pointer-events-none"
```

Comment:
You should add `zIndex 60` to the tailwind config and then use `z-60` instead of using scss.

https://tailwindcss.com/docs/z-index#customizing-your-theme

Reply:
Maybe this whole notification mechanic should be handled the same as modals, with teleport to a wrapper outside of the application.

https://vuejs.org/guide/built-ins/teleport.html

> The teleport to target must be already in the DOM when the <Teleport> component is mounted. Ideally, this should be an element outside the entire Vue application. If targeting another element rendered by Vue, you need to make sure that element is mounted before the <Teleport>.

Reply:
Yes good point!